src/trusted/service_runtime/sel_addrspace.c

```diff
@@ -186,6 +188,7 @@ NaClErrorCode NaClMemoryProtection(struct NaClApp *nap) {
                  NaClSysToUser(nap, start_addr) >> NACL_PAGESHIFT,
                  region_size >> NACL_PAGESHIFT,
                  NACL_ABI_PROT_NONE,
+                 NACL_ABI_PROT_NONE,
```

Comment:
Why do we sometimes use the NaCl ABI constants and sometimes the host ABI constants?
We should be consistent and probably use the NaCl ones.

Reply:
PROT_RW is defined the same way, yes. the PROT_NONE here is from the original google code it seems (remove histroy commit). Don't think we need to worry about that.

Reply:
Ok that's fine